README.md

```diff
@@ -36,20 +42,38 @@ This will likely cause significant changes to the environment, and should normal
    * Use `osx` or `linux` for `<platform>`
 1. Activate the environment: `source activate pinned_<date>`
 1. Check that the list of packages in the environment are as expected
-1. Export the environment, over-writing the previous `packages` file: `conda env export > conda3_packages-<ARCH>-64.yml`
-1. Update the `requirements.txt` file in the [linting repository](https://github.com/lsst/linting) to match the version numbers specified in the new conda environment.
```

Comment:
Doesn't this still hold, for whatever is listed there? (Only flake8 today, but maybe more later.)